doc/profiler.rst

```diff
@@ -51,6 +85,23 @@ Therefore, if ``print()`` called nothing else itself, its :guilabel:`Total Time`
 Meanwhile, :guilabel:`Total Time` for ``print_wrapper()`` would be 3 ms, but :guilabel:`Local Time` only 1 ms as the rest of that time was spend inside the ``print()`` function it called.
 
 
+
+================
+Profiler Plugins
+================
+
+There are two additional components that you can install to enable different types of profiling in Spyder. 
```

Comment:
```suggestion
There are two additional third-party plugins that you can install to enable other types of profiling in Spyder. 
```

* I think we should call them `plugins` instead of `components`.
* Improve phrasing a bit.